app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        current_command, file, new_file = command.split()
+        if file != new_file and current_command == "cp":
+            with open(file) as file, open(new_file, "a+") as new_file:
+                new_file.write(file.read())
+    except (ValueError, FileNotFoundError, PermissionError):
+        raise
```

Comment:
gj, but why do u need to catch error to reraise them ?)

Reply:
Okay, I fixed this => To return string with message. But we have moment when command "cp" not found in our function parameter, so for this code without return False, what the user should think or must do?

Reply:
if we talk about function, i think it is better to return string with message, or u can raise an error with message, or smth like this 
it seems to me, that False returning is not enough descriptive for user to understand, that command is incorrect )